tests/keras/wrappers/scikit_learn_test.py

```diff
@@ -89,6 +92,24 @@ def assert_classification_works(clf):
     assert np.allclose(np.sum(proba, axis=1), np.ones(num_test))
 
 
+def assert_string_classification_works(clf):
+    str_y_train = np.array(['cls1', 'cls2', 'cls3'])[y_train]
```

Comment:
This is too brittle IMO.  If someone changes num_class above then the test will start failing.  Try generating the string array based on num_class instead.